molecule/packages-dynamic/group_vars/all

```diff
@@ -13,3 +13,4 @@ pulp_pkg_repo: "https://yum.theforeman.org/pulpcore/3.14/el{{ ansible_distributi
 # doesn't:
 pulp_pkg_upgrade_all: true
 pulp_webserver_server: apache
+pulpcore_version: "3.14.5"
```

Comment:
So the design of packages mode is to by default, install the pulpcore version in the repo. By the default, the repo determines the versions of everything, plugins included.

This comment applies to all the other packages group_vars as well.

In fact, the minor version is in the URL of the repo.